Sources/Analytics/AnalyticsAction.swift

```diff
@@ -41,11 +41,25 @@ import SwiftUI
 ///
 public struct AnalyticsAction: Sendable {
     /// The current set of values
-    public let values: AnalyticsValues
+    public private(set) var values: AnalyticsValues
 
     /// Logs the specified `event`
-    /// - Parameter event: The `event` to log
-    public func callAsFunction(_ event: AnalyticsEvent, appending values: AnalyticsValues? = nil) {
+    /// - Parameters:
+    ///    - event: The `event` to log
+    ///    - appending: Any extra `values` to log with the existing `AnalyticsValues`
+    public mutating func callAsFunction(_ event: AnalyticsEvent, appending newValues: AnalyticsValues? = nil) {
```

Comment:
Just realising, you can't make it mutating otherwise all other calls to this function will gain the 'appended` values which would not be expected behaviour. The idea of this function is to append the values to _this_ specific log **only**